plover/assets/main.json

```diff
@@ -40939,7 +40942,7 @@
 "KP*P": "{&X}",
 "KP*P/PH*EPB": "X-Men",
 "KP*P/PWOBGS": "Xbox",
-"KP*P/PWOBGS/3-60": "Xbox 360",
+"KP*P/PWOBGS/306": "Xbox 360",
```

Comment:
O wait, `KP*P/PWOBGS/3-60` doesn't need its own stroke, right? That's just stroking Xbox and then the numbers 360 separately.